code/game/objects/items/bait.dm

```diff
@@ -116,3 +116,10 @@
 						/mob/living/simple_animal/hostile/retaliate/troll = 5,
 						/mob/living/simple_animal/hostile/retaliate/troll/bog = 5,
 						/mob/living/simple_animal/hostile/retaliate/troll/caerbannog = 2.5)
+
+/obj/item/bait/forestdelight
+	name = "meat wrapped in strange herbs"
+	icon_state = "baitbriar"
+	desc = "a piece of rotten and rancid meat wrapped in several herbs. The aroma induces both vomit and a nice herbal odor"
+	attracted_types = list (/mob/living/simple_animal/hostile/retaliate/mole/briars = 50,
```

Comment:
you should probably make it rarer and add normal moles to the poll

Reply:
Same reasoning as above. Plus, it litteraly takes more effort and time and rng than completing shrines + blood offering.  I think putting moles here is somewhat unfair since you will mostlikely have to kill or tame it, its just another rng for reason of rng. I would concider 4th crafting ingredient however.

Reply:
how is it unfair ?  you can obtain all the materials required in minutes . a volf gives you enought for 2 baits... i seen a lot of briars ingame with the moles very early into the round. i am not saying it should be superlategame , just not so easy that a single briar can fill the woods with them .
havent seen the in combat but looking at the stats they may be probably strong as hell